webpack.common.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+const path = require('path');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const webpack = require('webpack');
+
+/*::
+type Target = {|
+    name: string, // the name of output JS/CSS
+    entry: string, // the path to the entry point
+    library?: string // the name of the exported module
+|};
+*/
+
+/**
+ * List of targets to build
+ */
+const targets /*: Array<Target> */ = [
+    {
+        name: 'katex',
+        entry: './katex.webpack.js',
+        library: 'katex',
+    },
+    {
+        name: 'contrib/auto-render',
+        entry: './contrib/auto-render/auto-render.js',
+        library: 'renderMathInElement',
+    },
+    {
+        name: 'contrib/copy-tex',
+        entry: './contrib/copy-tex/copy-tex.js',
```

Comment:
I assume you left of the `library` property for this and `mathtex-script-type` since we weren't exporting the right thing in the browserify config which implies nobody was using it.  Is that correct?

Reply:
@kevinbarabash Yes, but also because `copy-tex` doesn't export anything and just adds a copy event listener to the document, so only its side effect is needed.